### PR TITLE
Dfcc 1250 remove nuspec

### PR DESCRIPTION
**Pull request by SimonWren**

### CODE REVIEW

File: DFC.TestAutomation.UI/DFC.TestAutomation.UI.csproj [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
-    <IncludeSymbols>false</IncludeSymbols>
+    <IncludeSymbols>true</IncludeSymbols>
     <SymbolPackageFormat>snupkg</SymbolPackageFormat>
     <AssemblyName>DFC.TestAutomation.UI</AssemblyName>
     <OutputType>Library</OutputType>
-    <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
     <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
-    <Version>1.0.0</Version>
+    <Version>2.0.1</Version>
```
Comment (jbarnes16503) [inline DFC.TestAutomation.UI/DFC.TestAutomation.UI.csproj:12]: May as well undo this change as it is overwritten by the pipeline.

File: DFC.TestAutomation.UI/DFC.TestAutomation.UI.csproj [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
     <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <ProjectGuid>12B1A410-DE8B-4F45-97E1-E0B502FDECC7</ProjectGuid>
-    <IncludeSymbols>false</IncludeSymbols>
+    <IncludeSymbols>true</IncludeSymbols>
     <SymbolPackageFormat>snupkg</SymbolPackageFormat>
     <AssemblyName>DFC.TestAutomation.UI</AssemblyName>
     <OutputType>Library</OutputType>
-    <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
```
Comment (jbarnes16503) [inline DFC.TestAutomation.UI/DFC.TestAutomation.UI.csproj:10]: This caused issues on the pipeline a few weeks back and we had to set this to false. I think however, this was only a problem because we wanted to use the nuspec and this was essentially superseding the nuspec. Just an FYI - no action needed.

File: DFC.TestAutomation.UI/DFC.TestAutomation.UI.csproj [hunk 3]
```diff
@@ -1,14 +1,15 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <ProjectGuid>12B1A410-DE8B-4F45-97E1-E0B502FDECC7</ProjectGuid>
-    <IncludeSymbols>false</IncludeSymbols>
+    <IncludeSymbols>true</IncludeSymbols>
```
Comment (jbarnes16503) [inline DFC.TestAutomation.UI/DFC.TestAutomation.UI.csproj:6]: This caused an issue on the pipeline but I can't recall what. Just a heads up. No action needed.

File: DFC.TestAutomation.UI/DFC.TestAutomation.UI.csproj [hunk 4]
```diff
@@ -1,14 +1,15 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```
Comment (jbarnes16503) [inline DFC.TestAutomation.UI/DFC.TestAutomation.UI.csproj:4]: Interesting. What does this do?

**Review by jbarnes16503 — CHANGES_REQUESTED** [review]

File: DFC.TestAutomation.UI/DFC.TestAutomation.UI.csproj (hunk 1 above)
Comment (SimonWren) [reply DFC.TestAutomation.UI/DFC.TestAutomation.UI.csproj:12]: gone

File: DFC.TestAutomation.UI/DFC.TestAutomation.UI.csproj (hunk 4 above)
Comment (SimonWren) [reply DFC.TestAutomation.UI/DFC.TestAutomation.UI.csproj:4]: it will force us to address any warnings in the build. will fail instead of warn.  Happy to remove, but I think it seem like good practice

File: DFC.TestAutomation.UI/DFC.TestAutomation.UI.csproj (hunk 3 above)
Comment (SimonWren) [reply DFC.TestAutomation.UI/DFC.TestAutomation.UI.csproj:6]: thumbs up emoji.  It would be good if this can be utilised. Will look into it if it causes a failure

File: DFC.TestAutomation.UI/DFC.TestAutomation.UI.csproj (hunk 2 above)
Comment (SimonWren) [reply DFC.TestAutomation.UI/DFC.TestAutomation.UI.csproj:10]: I'm not sure if the pipeline nuget pack requires this, but other nugets I've inspected use it.  Will investigate if issues arise

**Review by jbarnes16503 — APPROVED** [review]
